### PR TITLE
Rename internal USE flag media-gfx/graphicsmagick[imagemagick] -> media-gfx/graphicsmagick[imagemagick-compat]

### PR DESCRIPTION
**Pull request by SoapGentoo**
`media-gfx/graphicsmagick` should not use `USE="imagemagick"` in order to install imagemagick symlinks. This leads to cases where `USE="imagemagick graphicsmagick"` causes a USE conflict, which is unnecessary. See also https://bugs.gentoo.org/show_bug.cgi?id=590280

- Fixed some QA and bugs on the way.
- Also ported some ebuilds to `python-r1`

### CODE REVIEW

File: x11-plugins/wmudmount/wmudmount-2.2-r2.ebuild [hunk 1]
```diff
@@ -1,17 +1,17 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
-inherit eutils gnome2-utils
+EAPI=6
+inherit eutils gnome2-utils xdg
```
Comment (mgorny) [inline x11-plugins/wmudmount/wmudmount-2.2-r2.ebuild]: Why u inherit xdg then don't call it's pre/postinst/rm?

Comment (mgorny) [inline x11-plugins/wmudmount/wmudmount-2.2-r2.ebuild]: Oh, you did double revbump and confused github ;-p.

**Comment by monsieurp** [comment]
@SoapGentoo ping

**Comment by monsieurp** [comment]
pong. hi?

**Comment by SoapGentoo** [comment]
@monsieurp I'll reopen it in due time.

**Pull request closed without merging**